### PR TITLE
If the current link href is empty, use the current window location href

### PR DESCRIPTION
**Pull request by tommymarshall**
This fixes an issue that @awavering was having.

### CODE REVIEW

File: trackomatic.js [hunk 1]
```diff
@@ -342,7 +342,7 @@ function Trackomatic(tracker, config) {
           return url.replace(/^https?:\/\//, '').split('/')[0]
         }
         var getVisitData = function(event, link) {
-          var url           = link.href
+          var url           = link.href === "" ? window.location.href : link.href
```
Comment (solomonhawk) [inline trackomatic.js]: How about `var url = link.href || window.location.href`?

**Comment by solomonhawk** [comment]
:+1:

File: trackomatic.js (hunk 1 above)
Comment (averyvery) [inline trackomatic.js]: True, a blank string is falsey....

Comment (tommymarshall) [inline trackomatic.js]: ah empty string if falsey. natch.

**Comment by averyvery** [comment]
:+1:

**Pull request merged**